components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.ts

```diff
@@ -30,21 +34,30 @@ export class ServicesSidebarComponent implements OnInit {
 
     this.serviceGroupName$ = this.store.select(createSelector(serviceGroupState,
       (state) => state.selectedServiceGroupName));
+
+    this.currentPage = 1;
   }
 
   public closeServicesSidebar() {
     this.closeServicesSidebarEvent.emit(null);
   }
 
-  public updateServicesFilters(health: string): void {
+  public updateServicesFilters(health: string, page?: number): void {
     this.selectedHealth = health;
     const servicesFilters: ServicesFilters = {
       service_group_id: this.serviceGroupId,
-      health: health
+      health: health,
+      page: page,
```

Comment:
If `page` here will continue to be optional, can we use `this.currentPage` instead to be sure we will have a page?